app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,8 @@
 <resources>
     <string name="app_name">Movie Fiend</string>
+    <string name="poster">poster</string>
+    <string name="more_details">more details</string>
+    <string name="tmdb_now_playing_url">http://api.themoviedb.org/3/movie/now_playing?api_key=%s</string>
```

Comment:
Add `translatable="false"` to indicate that this string should not be localized.
Especially important in Venice where there are localization teams that translate the strings added to the project.